app/views/RoomInfoView/components/UserInfoIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Text } from 'react-native';
+import { BorderlessButton } from 'react-native-gesture-handler';
+
+import { CustomIcon, TIconsName } from '../../../containers/CustomIcon';
+import styles from '../styles';
+import { useTheme } from '../../../theme';
+import { useVideoConf } from '../../../lib/hooks/useVideoConf';
+import i18n from '../../../i18n';
+import { useAppSelector } from '../../../lib/hooks';
+import { compareServerVersion } from '../../../lib/methods/helpers';
+
+// TODO: change other icons on future
```

Comment:
Can you create tasks to do it and remove the comment?
I just read a comment like this asking to remove some logic after we update to RN 0.65.
We're on 0.68.6 and the comment lost meaning 😂